keyform/models.py

```diff
@@ -2,5 +2,60 @@
 from __future__ import unicode_literals
 
 from django.db import models
+from django.conf import settings
+from django.utils.timezone import now
 
-# Create your models here.
+
+class Request(models.Model):
+
+    REQUEST_TYPES = (
+        ('bk', 'Broken Key'),
+        ('sk', 'Stolen Key'),
+        ('wk', 'Worker Key'),
+    )
+
+    PAYMENT_TYPES = (
+        ('ca', 'Cash'),
+        ('ch', 'Check'),
+    )
+
+    building = models.ForeignKey('Building')
+    student_name = models.CharField(max_length=128)
+    reason_for_request = models.CharField(max_length=2, choices=REQUEST_TYPES)
+    amt_recieved = models.DecimalField(max_digits=7, decimal_places=2, default=0, blank=True)
+    paid_by = models.CharField(max_length=2, choices=PAYMENT_TYPES)
+    charge_amount = models.DecimalField(max_digits=7, decimal_places=2)
+    staff_name = models.ForeignKey(settings.AUTH_USER_MODEL)
```

Comment:
Maybe this field should  just be "staff" since it points to the staff user object, and not just the staff name